doc/nvim-treesitter.txt

```diff
@@ -128,6 +128,18 @@ You can use |:TSInstall| `all` to install all parsers.
 
 List informations about currently installed parsers
 
+							           *:TSUpdate*
+:TSUpdate {language}~
+
+Update the installed parser of {language} or all installed parsers
+if {language} is omitted.
+
+							        *:TSUninstall*
+:TSUninstall {language}~
```

Comment:
shouldn't these be on the same line ?

Reply:
If they are in the same line and there is a long function everything will need to be reformatted, so it's easier to maintain in this way.